mypy/server/update.py

```diff
@@ -764,7 +768,7 @@ def replace_modules_with_new_variants(
 
 
 def propagate_changes_using_dependencies(
-        manager: BuildManager,
+        manager: FineGrainedBuildManager,
```

Comment:
This couples the function more tightly with `FineGrainedBuildManager` and performs additional mutation that is not visible in the interface of the function. It would be better to pass `process_targets` as an extra argument (and preserve the old `manager` argument`). This might also reduce code churn.

Reply:
OK